### PR TITLE
swipl: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by kamahen — COMMENTED** [review]
You might want to add the `-q` option for "quiet" (not print out the banner.

Also, a slightly longer example might be helpful. E.g., if `/tmp/ex.pl` contains
```
p(word1).
p(word2).
```
then this command:
```
swipl -q -g 'forall(p(X), writeln(X))' -g halt /tmp/abc.pl
```
produces this output:
```
word1
word2
```
Also:
```
$ swipl -q -t 'writeln(hello-world)' 
hello-world
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/swipl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# swipl
+
+> SWI-Prolog a comprehensive free Prolog environment.
```
Comment (marchersimon) [inline pages/common/swipl.md:3]: Whatever you prefer:
```suggestion
> SWI-Prolog - A comprehensive free Prolog environment.
```
```suggestion
> SWI-Prolog: a comprehensive free Prolog environment.
```

File: pages/common/swipl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.swi-prolog.org/>.
+
+- Start an interactive session:
+
+`swipl`
+
+- Execute a command:
```
Comment (marchersimon) [inline pages/common/swipl.md:10]: If we include the `--quiet` flag, we also have to explain it.
```suggestion
- Execute a command without showing any output:
```

File: pages/common/swipl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`swipl --quiet -t "{{command}}"`
+
+- Execute a script:
+
+`swipl {{path/to/file.pl}}`
+
+- Print shell configuration variables:
```
Comment (marchersimon) [inline pages/common/swipl.md:18]: This really improves the reading:
```suggestion
- Print all shell configuration variables:
```

**Review by marchersimon — APPROVED** [review]
Thanks @EmilySeville7cfg

**Pull request merged**